tools/mason/MasonSearch.chpl

```diff
@@ -123,9 +147,17 @@ proc consumeArgs(ref args : [] string) {
   assert(sub == "search");
   args.pop_front();
 
-  const options = {"--no-update", "--debug"};
+  const options = {"--no-update", "--debug", "--show"};
 
   while args.size > 0 && options.contains(args.head()) {
     args.pop_front();
   }
 }
+
+proc showToml(tomlFile : string, packageName : string) {
+  const brickPath = MASON_HOME +'/mason-registry/Bricks/' +packageName + '/' + tomlFile;
+  const openFile = openreader(brickPath);
+  const toml = parseToml(openFile);
```

Comment:
Let's make this an owned toml so that we don't leak any memory. Also, don't forget to close our reader we opened in the line above!